vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -490,13 +490,13 @@ def build_template_grid(templates,sort_order="DESC",sort_by="name")
   def dialog_partial_for_workflow
     case (@edit || @options).try(:[], :wf)
     when MiqProvisionVirtWorkflow
-      "prov_dialog"
+      "miq_request/prov_dialog"
     when MiqProvisionConfiguredSystemForemanWorkflow
-      "prov_configured_system_foreman_dialog"
+      "miq_request/prov_configured_system_foreman_dialog"
     when MiqHostProvisionWorkflow
-      "prov_host_dialog"
+      "miq_request/prov_host_dialog"
     when VmMigrateWorkflow
-      "prov_vm_migrate_dialog"
+      "miq_request/prov_vm_migrate_dialog"
```

Comment:
If we have ruby classes here, why not put a dialog_partial method on each and eliminate the case statement?